Terminal.Gui/Views/Menu.cs

```diff
@@ -1296,11 +1299,11 @@ internal void Activate (int idx, int sIdx = -1, MenuBarItem subMenu = null)
 				previousFocused = SuperView == null ? Application.Current.Focused : SuperView.Focused;
 
 			OpenMenu (idx, sIdx, subMenu);
-			if (!SelectEnabledItem (openCurrentMenu.barItems.Children, openCurrentMenu.current, out openCurrentMenu.current)
-				&& subMenu == null && !CloseMenu (false)) {
+			//if (!SelectEnabledItem (openCurrentMenu.barItems.Children, openCurrentMenu.current, out openCurrentMenu.current)
```

Comment:
Why comment this out vs. just deleting it?

Reply:
Ok I'll delete it.